unime/src/i18n/en-GB/index.ts

```diff
@@ -4,12 +4,27 @@ import en from '../en';
 import type { Translation } from '../i18n-types';
 
 const en_GB = extendDictionary(en, {
-  // ...(en as Translation),
-  // ONBOARDING: {
-  //   CUSTOMIZE: {
-  //     NAVBAR_TITLE: 'Customisation',
-  //   },
-  // },
+  ...(en as Translation),
+  ONBOARDING: {
+    CUSTOMIZE: {
+      NAVBAR_TITLE: 'Customisation',
+    },
+    PLEDGE: {
```

Comment:
I think there's something off with the nesting a little, the base structure is as follows:

```
PLEDGE: {
  ITEM_1: {
    DESCRIPTION: '...'
  },
  ITEM_2: {
    DESCRIPTION: '...'
  }
}
```

Your current changes would introduce:

```
PLEDGE: {
  DESCRIPTION: '...'
}
```

Solution: add a wrapping `ITEM_1` around the first `DESCRIPTION`.